src/makemon.c

```diff
@@ -354,9 +354,9 @@ m_initweap(register struct monst *mtmp)
             while (!rn2(3))
                 (void) mongets(mtmp, APPLE + rn2(CARROT - APPLE));
         } else if (is_dwarf(ptr)) {
-            if (!rn2((In_mines(&u.uz)) ? 10 : 30)) {
-                otmp = mongets(mtmp, POT_BOOZE;
-                otmp->quan = 3;
+            if (!rn2((In_mines(&u.uz)) ? 40 : 8)) {
```

Comment:
the parens around In_mines seem unneeded

Reply:
does NH have a magical program like this? https://github.com/crawl/crawl/blob/master/crawl-ref/source/util/checkwhite